src/payload/order/patchOrderPayload.ts

```diff
@@ -0,0 +1,20 @@
+import { PatchOrderStop } from "../../models/patchOrderStop";
+import PatchOrderPayloadBuilder from "./patchOrderPayloadBuilder";
+
+export default class PatchOrderPayload {
+    protected stops?: PatchOrderStop[];
+
+    constructor(builder: PatchOrderPayloadBuilder) {
+        const { stops } = builder;
+
+        if (!stops || !(stops?.length ?? 0 > 2)) {
```

Comment:
should we also check for the maximum stops?

Reply:
yes, just added